utils/src/test/scala/za/co/absa/enceladus/utils/transformations/ArrayTransformationsSuite.scala

```diff
@@ -68,13 +68,13 @@ class ArrayTransformationsSuite extends AnyFunSuite with TZNormalizedSparkTestBa
   test("Testing array transform") {
     val df = spark.createDataFrame(inputDataOrig)
 
-    val t = ArrayTransformations.arrayTransform(df, "vals")({
-      case d =>
+    val t = ArrayTransformations.arrayTransform(df, "vals"){
+      d =>
```

Comment:
```suggestion
    val t = ArrayTransformations.arrayTransform(df, "vals"){ d =>
```

just a formatting suggestion